### PR TITLE
add coveralls

### PR DESCRIPTION
**Pull request by JiejayLan**
Overview of PR
1.add coveralls
2.update travis-CI
3.update README

Technical Changes
1.add coveralls

Architecture Changes
TBD

Help!
TBD

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -8,6 +8,7 @@
     "lint": "eslint ./src",
     "lint:fix": "eslint ./src --fix",
     "test": "react-scripts test --watchAll=false",
+    "coveralls": "react-scripts test --coverage --watchAll=false && cat ./coverage/lcov.info | coveralls",
```
Comment (songrenzhao) [inline package.json]: What does this do? @JiejayLan

File: package.json [hunk 2]
```diff
@@ -38,6 +39,7 @@
   },
   "devDependencies": {
     "babel-eslint": "^10.0.2",
+    "coveralls": "^3.0.6",
```
Comment (songrenzhao) [inline package.json]: Same as above

Comment (JiejayLan) [reply package.json]: coveralls is similar to code climate

File: package.json (hunk 1 above)
Comment (JiejayLan) [reply package.json]: run the jest --coverage and then push the coverage report to my coveralls account

**Pull request merged**